src/parser.rs

```diff
@@ -5979,9 +5991,7 @@ impl<'a> Parser<'a> {
         } else if dialect_of!(self is BigQueryDialect | GenericDialect)
             && self.parse_keyword(Keyword::UNNEST)
         {
-            self.expect_token(&Token::LParen)?;
-            let expr = self.parse_expr()?;
-            self.expect_token(&Token::RParen)?;
+            let exprs = self.parse_unnest_exprs()?;
```

Comment:
I think the standard pattern for parsing exprs in sqlparser is like this https://github.com/sqlparser-rs/sqlparser-rs/blob/efd8cb7fd1c4e6654b101868f7a76d80f66da74b/src/parser.rs#L484-L488

So in this case like this (and can remove the `parse_unnest_exprs`)

```suggestion
            self.expect_token(&Token::LParen)?;
            let exprs = self.parse_comma_separated(Parser::parse_expr)?;
            self.expect_token(&Token::RParen)?;
```